src/styles/blocks/header.scss

```diff
@@ -0,0 +1,10 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
```

Comment:
Consider using variables for spacing values like padding to maintain consistency and make it easier to manage these values. For example, you could define a variable like $padding-horizontal at the top of your SCSS file and then use it here.